.config/gitpod/Dockerfile

```diff
@@ -12,11 +11,10 @@ ARG NPM_VERSION=9.6.7
 # Set a variable for the ungit version
 ARG UNGIT_VERSION=1.5.23
 
-# Update system and install packages, including Docker
-# Replace "python3.10" with "$(cut -d '=' -f 2 /tmp/python_version.config)" in the apt-get install command
+# Update system and install packages*
 # hadolint ignore=DL3008,DL3013
```

Comment:
@ThomasSanson 

Indeed, since the packages have already been installed, there is no need to keep them in the image.

You will find my correction in this commit: 5d42bf05a1105e1186cc6a1eddf6f7b771451332

Reply:
@AurelienToitot 

Due to your response, it appears possible that you may not have fully understood.

The DL3009 rule (DL indicating it comes directly from Hadolint) states:
Remove apt-get lists after installation.

In your multi-stage configuration, the purging of apt is not performed in the "builder" stage for Hadolint, which results in an error. However, you correctly perform the purge later in the "cleanup" stage.

Therefore, and simply because we are aware of the steps we have implemented, we can ignore this rule (DL3009) in the RUN command on line 17.

Indeed, we know that it is done later (the apt cache will be purged).